jerry-core/jerry.cpp

```diff
@@ -124,6 +124,242 @@ jerry_make_api_unavailable (void)
 } /* jerry_make_api_unavailable */
 
 /**
+ * Returns whether the given jerry_api_value_t is void.
+ */
+bool
+jerry_api_value_is_void (const jerry_api_value_t *value_p)
+{
+  return value_p->type == JERRY_API_DATA_TYPE_VOID;
+}
```

Comment:
Missing argument comments and function end comments (as per style), this should be:
```
bool
method_name (int arg) /** << my super comment on the argument */
{
...
} /* method_name */
```